pkg/cmd/access_checker.go

```diff
@@ -10,7 +13,7 @@ import (
 // IsAllowedTo checks whether the current user is allowed to perform the given action in the specified namespace.
 // Specifying "" as namespace performs check in all namespaces.
 type AccessChecker interface {
-	IsAllowedTo(verb, resource, namespace string) (bool, error)
+	IsAllowedTo(ctx context.Context, verb, resource, namespace string, opts metav1.CreateOptions) (bool, error)
```

Comment:
I would not change this interface because we're just passing `context.Background()` and an empty `metav1.CreateOptions{}` struct to `client.Create()` method